main/network.c

```diff
@@ -657,13 +657,13 @@ PHPAPI void php_network_populate_name_from_sockaddr(
 	}
 
 	if (textaddr) {
-		char abuf[256];
+		char abuf[INET6_ADDRSTRLEN];
```

Comment:
Is this available if `HAVE_IPV6` is false?

Reply:
yes it is independent from it.